gradle/common-java.gradle

```diff
@@ -32,7 +32,7 @@ if (java7JreDir) {
     def requiredJavaBootArchives = ["rt.jar", "jsse.jar"]
     def bootClasspath = ""
     requiredJavaBootArchives.each { a ->
-        def archivePath = new File(java7JreDir, "lib/$a")
+        def archivePath = new File(java7JreDir, "jre/lib/$a")
```

Comment:
the JAVA_JRE_7 dir should point to the JRE root. If you have the JDK, then you point it to JDK_HOME/jre. This breaks if you point it to the JRE and not the JDK.